App/Application/Members/SignUpMember.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\Members;
+
+final class SignUpMember
+{
+    public function __construct(
+        private readonly string $firstName,
+        private readonly string $lastName,
+        private readonly string $emailAddress,
+        private readonly string $password
+    ) {
+    }
+
+    /**
+     * @return string
+     */
+    public function getFirstName(): string
```

Comment:
Since all fields are `readonly` and you have only getters, why don't you get rid of `get` prefix and use shorter, and IMHO much better, accessors like `firstName()`? Or even make those fields public and use them directly, without introducing methods? They don't do anything more than accessing private property, it does not make sense 🤷‍♂️

Reply:
This is a great idea, I'll update all of the getter methods and remove the `get` prefix as you suggested. I'll have to think about the properties usage a bit later but it's a good suggestion as well.

Reply:
With public readonly property you have the same level of safety like with getter 🤷‍♂️ I would introduce methods only if there's additional logic, like concatenating name and surname into full name.

Reply:
I removed the getter methods and replaced with the public read only properties as you suggested. Thanks for the idea.